### DISCUSSION TITLE
Lambda with Quarkus connecting to AWS RDS Postgresql via RDS Proxy - how to configure?

### DISCUSSION THREAD

**Discussion by rafkoko** [quarkusio/quarkus#34757] [Q&A]
We are currently in the process of developing Lambdas using the Quarkus framework. Our plan is to utilize Panache ORM and establish a connection to AWS RDS Postgresql through RDS Proxy. However, we are encountering difficulties in finding suitable examples or guidelines to assist us in this endeavor. We would greatly appreciate it if you could provide us with a reliable guideline or a working example that aligns with our intended setup. It is worth mentioning that we have previous experience working with Spring and Hibernate, which could potentially aid in understanding the integration with Panache ORM.

[a comment by an automated account is not shown]

**Comment by gdelhaye** [comment]
Did you find a solution? We are currently in the same situation.
